lib/request-handler.js

```diff
@@ -22,6 +22,9 @@ const getCrossOriginHeader = (fragmentUrl, host) => {
  */
 module.exports = function processRequest (options, request, response) {
     this.emit('start', request);
+    this.index = 0;
+    this.maxStyleLinks = options.maxStyleLinks;
+    this.maxScriptLinks = options.maxScriptLinks;
```

Comment:
unless I'm reading it wrong, these variables are only used in this scope - so they don't really need to be bound to `this`..?